c16719140.lua

```diff
@@ -30,18 +30,17 @@ function c16719140.spcost(e,tp,eg,ep,ev,re,r,rp,chk)
 	e:SetLabel(100)
 	if chk==0 then return true end
 end
-function c16719140.rfilter(c)
-	return c:GetOriginalLevel()>0
-end
 function c16719140.costfilter(c,e,tp,mg,rlv)
+	if c:GetLevel()==0 then return false end
 	local lv=c:GetLevel()-rlv
-	return lv>0 and c:IsSetCard(0xed) and c:IsType(TYPE_MONSTER) and c:IsAbleToGraveAsCost()
-		and (c:IsCanBeSpecialSummoned(e,0,tp,false,false) or c:IsCanBeSpecialSummoned(e,0,tp,false,false,POS_FACEDOWN_DEFENSE))
-		and mg:CheckWithSumGreater(Card.GetOriginalLevel,lv)
+	if c:IsSetCard(0xed) and c:IsType(TYPE_MONSTER) and c:IsAbleToGraveAsCost()
+		and (c:IsCanBeSpecialSummoned(e,0,tp,false,false) or c:IsCanBeSpecialSummoned(e,0,tp,false,false,POS_FACEDOWN)) then
+		return (lv<0 and mg:GetCount()>0) or mg:CheckWithSumGreater(Card.GetOriginalLevel,lv)
```

Comment:
I think it should be
lv<=0
I doubt that the SumGreater function returns a true value if lv==0 or am I wrong here^^